manifests/join.pp

```diff
@@ -33,6 +34,11 @@
     } else {
       $ad_join_domain_controller_command = ''
     }
+    if $ad_join_service_names != [] {
+        $ad_join_service_names_comand = join([" --service-name='", join($ad_join_service_names, "' --service-name='"), "'"], '')
```

Comment:
please correct and spell command properly

Reply:
comand has been corrected to command